src/m_misc.c

```diff
@@ -2273,31 +2276,40 @@ default_t defaults[] = {
   {
     "midi_player",
     (config_t *) &midi_player, NULL,
-    {0}, {0, MAX_MIDI_PLAYERS},
+    {0}, {0, 2},
     number, ss_gen, wad_no,
 #if defined(_WIN32)
-    "MIDI player"
+    "0 for Native (default), "
 #else
     "0 for SDL2 (default), "
-  #if defined(HAVE_FLUIDSYNTH)
+#endif
+#if defined(HAVE_FLUIDSYNTH)
     "1 for FluidSynth, 2 for OPL Emulation"
-  #else
+#else
     "1 for OPL Emulation"
-  #endif
 #endif
   },
 
+  {
+    "midi_player_menu",
+    (config_t *) &midi_player_menu, NULL,
+    {0}, {0, MAX_MIDI_PLAYERS - 1}, number, ss_gen, wad_no,
+    "MIDI Player menu index"
+  },
+
 #if defined(HAVE_FLUIDSYNTH)
+  {
+    "soundfont_dir",
+    (config_t *) &soundfont_dir, NULL,
+    {.s = "soundfonts"}, {0}, string, ss_none, wad_no,
```

Comment:
May I commit the little change that is required to support Linux directly to your branch?

Reply:
Sure!